terraform/dev/ingress.tf

```diff
@@ -0,0 +1,30 @@
+resource "kubernetes_ingress_v1" "general_ingress" {
+  metadata {
+    name = "general-ingress"
+    annotations = {
+      "alb.ingress.kubernetes.io/scheme"          = "internet-facing"
+      "alb.ingress.kubernetes.io/subnet"          = "subnet-a3f202fa, subnet-fc843999, subnet-0fc91a78"
```

Comment:
Since these are in multiple places, is there a way for us to set them as constants somewhere? I'm imagining that we might want to update them at some point.

Reply:
(Oh, maybe in the terraform variables file?)

Reply:
Good idea